File: coding-test/src/ldh/프로그래머스/day241114/가운데_글자_가져오기/solution.js
function solution(s) {
    let c = s.length;

    if (c % 2 === 1) {
        return s.charAt(Math.floor(c / 2));
    } else {
        return s.substring(c / 2 - 1, c / 2 + 1);
    }
}

File: coding-test/src/ldh/프로그래머스/day241114/가운데_글자_가져오기/Solution.java
package ldh.프로그래머스.day241114.가운데_글자_가져오기;

class Solution {
    public String solution(String s) {
        int length = s.length();
        int mid = length / 2;
        System.out.println(mid);

        if (length % 2 == 1) {
            return s.substring(mid, mid + 1);
        }
        else {
            return s.substring(mid - 1, mid + 1);
        }

    }
}

